src/thirdparty/tests/CMakeLists.txt

```diff
@@ -27,20 +29,20 @@ if ( UMPIRE_FOUND )
 
 endif()
 
-########################################
+#------------------------------------------------------------------------------
 # Smoke test for RAJA thirdparty library
-########################################
+#------------------------------------------------------------------------------
 if ( RAJA_FOUND )
 
   set( raja_smoke_dependencies raja gtest )
 
-  blt_list_append( TO raja_smoke_dependencies ELEMENTS openmp IF ENABLE_OPENMP )
+  blt_list_append( TO raja_smoke_dependencies ELEMENTS openmp IF ${ENABLE_OPENMP} )
 
-  blt_list_append( TO raja_smoke_dependencies ELEMENTS umpire IF UMPIRE_FOUND )
+  blt_list_append( TO raja_smoke_dependencies ELEMENTS umpire IF ${UMPIRE_FOUND} )
```

Comment:
This may not be equivalent due to UMPIRE_FOUND being undefined if not found. CMake would evaluate it here instead of inside the macro.

Reply:
Thanks @white238 -- I'll look into it.

Reply:
Looks like it works either way.  
Specifically, the element is added if the variable is defined and evaluates to ``TRUE``, regardless of whether you escape it or not.

I reverted these changes since not escaping will likely retain a bit more context and possibly produce better error messages.

Reply:
I created a BLT PR for this test: https://github.com/LLNL/blt/pull/322